src/lib.rs

```diff
@@ -260,7 +260,7 @@ pub struct CliOpts {
         name = "NETWORK",
         short = "n",
         long = "network",
-        default_value = "testnet"
+        default_value = "regtest"
```

Comment:
I didn't see this change before, sorry but I don't think we should make "regtest" the default network. Most casual users will expect "testnet" since that's how existing docs and blogs are written.

Reply:
Ya I thought about that too. But to me it occurred that bdk-cli is not for casual user but for devs trying out their own bdk wallet setup. I find myself setting the network to regtest all the time. Because casual testing is harder with testnet. Also the `BitcoinD` backend crate defaults to regtest. It would be messy if we try to make it testnet and then delete temporary datadir after each execution.. Testnet takes a long time to sync. Which is not suitable for casual testing. 

I feel testnet is useful when devs already have a prototype working and want to simulate mainnet type behavior without real money.

So does this make a good justification to make the default to regtest? We can change the docs and other instructions at other places..

Reply:
How about for now we leave testnet.. but once this easy regtest node setup is ready for prime time we make regtest the default? That will also give us time to update the docs before it takes effect.

Reply:
Yes makes sense..